p2p-lib/src/main/java/org/smartregister/p2p/search/contract/P2PManagerListener.kt

```diff
@@ -28,4 +29,8 @@ interface P2PManagerListener : WifiP2pManager.ConnectionInfoListener {
     fun handleAccessFineLocationNotGranted()
 
     fun handleMinimumSDKVersionNotMet(minimumSdkVersion: Int)
+
+    fun showP2PSelectPage(deviceRole: DeviceRole, deviceName: String)
```

Comment:
I think we can have these two in a different interface. This interface is for P2pManager events